data/svgs/README.md

```diff
@@ -90,6 +90,7 @@ as of the time of import. See the libratbag COPYING file for details.
  logitech-g900.svg
  logitech-mx-anywhere2.svg
  logitech-mx-anywhere2s.svg
+ logitech-mx-ergo.svg
```

Comment:
can you drop this hunk please? this is just the list of files imported from libratbag a while ago, this doesn't apply for this one.